bucket/qv2ray.json

```diff
@@ -0,0 +1,53 @@
+{
+    "version": "2.2.3",
```

Comment:
Qv2ray has updated to [`2.3.1`](https://github.com/Qv2ray/Qv2ray/releases/v2.3.1).
We shall consider updating this.